polaris-service/src/test/java/org/apache/polaris/service/PolarisApplicationIntegrationTest.java

```diff
@@ -103,6 +106,11 @@
   SnowmanCredentialsExtension.class
 })
 public class PolarisApplicationIntegrationTest {
+  @TempDir private static Path tempDir;
+  private static Supplier<String> CURRENT_LOG = () -> tempDir.resolve("application.log").toString();
+  private static Supplier<String> ARCHIVED_LOG =
+      () -> tempDir.resolve("application-%d-%i.log.gz").toString();
```

Comment:
Looks like the archive logs are not used anywhere. Do we need this change?

Reply:
I just cleaned this up.  We don't need 'archivedLogFilenamePattern' or ARCHIVED_LOG